etc/linux-upstart/system/syncthing.conf

```diff
@@ -0,0 +1,13 @@
+description "Syncthing"
+
+start on (local-filesystems and net-device-up IFACE!=lo)
+stop on runlevel [!2345]
+
+env STNORESTART=yes
+env HOME=/home/jake
```

Comment:
Yes, it should be $USER.



On Oct 24, 2015, 6:08 AM, at 6:08 AM, Audrius Butkevicius <notifications@github.com> wrote:
>> @@ -0,0 +1,13 @@
>> +description "Syncthing"
>> +
>> +start on (local-filesystems and net-device-up IFACE!=lo)
>> +stop on runlevel [!2345]
>> +
>> +env STNORESTART=yes
>> +env HOME=/home/jake
>
>Can this be $USER?
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/syncthing/syncthing/pull/2405/files#r42933334